integration_tests/src/main/python/data_gen.py

```diff
@@ -583,20 +583,13 @@ class TimestampGen(DataGen):
     """Generate Timestamps in a given range. All timezones are UTC by default."""
     def __init__(self, start=None, end=None, nullable=True, tzinfo=timezone.utc):
         super().__init__(TimestampNTZType() if tzinfo==None else TimestampType(), nullable=nullable)
+        min_start = datetime(1, 1, 1, tzinfo=tzinfo)
```

Comment:
`min_start` seems to be used only once below. So why don't just:
```
if start is None:
    start = datetime(1, 1, 1, ...)
```

Reply:
Fixed.